`TestAccEKSCluster_version` failing with `InvalidParameterException` (kubernetes version)

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform CLI and Terraform AWS Provider Version

<!--- Please run `terraform -v` to show the Terraform core version and provider version(s). If you are not running the latest version of Terraform or the provider, please upgrade because your issue may have already been fixed. [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions). --->

### Test output:
```
=== RUN   TestAccEKSCluster_version
=== PAUSE TestAccEKSCluster_version
=== CONT  TestAccEKSCluster_version
    cluster_test.go:211: Step 1/3 error: Error running apply: exit status 1

        Error: error creating EKS Cluster (tf-acc-test-5451735558585737152): InvalidParameterException: unsupported Kubernetes version
        {
          RespMetadata: {
            StatusCode: 400,
            RequestID: "1cbf0ae2-7bd9-4e38-85a4-3084bdbb632b"
          },
          ClusterName: "tf-acc-test-5451735558585737152",
          Message_: "unsupported Kubernetes version"
        }

          with aws_eks_cluster.test,
          on terraform_plugin_test.tf line 59, in resource "aws_eks_cluster" "test":
          59: resource "aws_eks_cluster" "test" {

--- FAIL: TestAccEKSCluster_version (15.94s)
FAIL
```

Code in question:
https://github.com/hashicorp/terraform-provider-aws/blob/7fc13cccad22150d502110e7aa74430fccd7765b/internal/service/eks/cluster_test.go#L218-L236

Reference:
* https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

--- a/internal/service/eks/cluster_test.go
+++ b/internal/service/eks/cluster_test.go
@@ -215,10 +215,10 @@ func TestAccEKSCluster_version(t *testing.T) {
 		CheckDestroy: testAccCheckClusterDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccClusterConfig_Version(rName, "1.16"),
+				Config: testAccClusterConfig_Version(rName, "1.19"),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckClusterExists(resourceName, &cluster1),
-					resource.TestCheckResourceAttr(resourceName, "version", "1.16"),
+					resource.TestCheckResourceAttr(resourceName, "version", "1.19"),
 				),
 			},
 			{
@@ -227,11 +227,11 @@ func TestAccEKSCluster_version(t *testing.T) {
 				ImportStateVerify: true,
 			},
 			{
-				Config: testAccClusterConfig_Version(rName, "1.17"),
+				Config: testAccClusterConfig_Version(rName, "1.20"),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckClusterExists(resourceName, &cluster2),
 					testAccCheckClusterNotRecreated(&cluster1, &cluster2),
-					resource.TestCheckResourceAttr(resourceName, "version", "1.17"),
+					resource.TestCheckResourceAttr(resourceName, "version", "1.20"),
 				),
 			},
 		},

